File: ftgo-restaurant-service/src/main/java/me/jangjunha/ftgo/restaurant_service/web/WebConfigurer.java
package me.jangjunha.ftgo.restaurant_service.web;

import me.jangjunha.ftgo.common.web.AuthContextArgumentResolver;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.method.support.HandlerMethodArgumentResolver;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

import java.util.List;

@Configuration
public class WebConfigurer implements WebMvcConfigurer {
    @Override
    public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
        resolvers.add(new AuthContextArgumentResolver());
    }
}

File: ftgo-consumer-service/src/main/kotlin/me/jangjunha/ftgo/consumer_service/web/WebConfigurer.kt
package me.jangjunha.ftgo.consumer_service.web

import me.jangjunha.ftgo.common.web.AuthContextArgumentResolver
import org.springframework.context.annotation.Configuration
import org.springframework.web.method.support.HandlerMethodArgumentResolver
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

@Configuration
class WebConfigurer: WebMvcConfigurer {
    override fun addArgumentResolvers(resolvers: MutableList<HandlerMethodArgumentResolver>) {
        resolvers.add(AuthContextArgumentResolver())
    }
}
